Automattic/wp-calypso
# Guided Transfer: UI for guided transfer completion screen

This improves on the Guided Transfer post-payment screen where the Guided Transfer is underway.

Note that this PR will affect **both** the existing live wp-admin guided transfer flow, as well as the forthcoming Calypso guided transfer flow as they both use the Calypso checkout.

A few deviations from the original design:

 - The site name isn't currently shown in the blue thank-you header. This isn't possible currently as the receipt object doesn't yet contain info to link the purchase to a site. A server-side change should fix this.
 - The "What now?" header isn't in the original design, but seems to be a default for the thank you page.

| Before | After |
|-----|-----|
| ![screen shot 2016-06-28 at 8 52 14 pm](https://cloud.githubusercontent.com/assets/416133/16412944/3d898f12-3d72-11e6-89f7-b1a003e3dabd.png) | ![screen shot 2016-06-29 at 8 50 30 am](https://cloud.githubusercontent.com/assets/416133/16435047/aae8967a-3dd6-11e6-9083-6a213d554548.png) |

### Some questions

**Current Flow:** wp-admin > Guided Transfer > Purchase > **Thank-you page** > Enter host details
**Future Flow:** Calypso > Guided Transfer > Enter host details > Purchase > **Thank-you page**

In the current wp-admin flow, we ask the user to purchase the guided transfer **before** we grab their target host's login details. This thankyou screen makes it appear that the user doesn't need to do anything further, which is true for the future Calypso flow. However for the wp-admin flow that's currently in use, I'm wondering if we should add an extra box at the top - for example something like:

-----------

**Do we have your new host details?**
So that we can transfer your site, we'll need your login details for the new host. Head over to [wp-admin](https://testsite2982.wordpress.com/wp-admin/paid-upgrades.php?page=guided-transfer) to get started.

---------

### How to test

1. Open a test site's wp-admin, go to **Tools** > **Export** > Purchase Guided Transfer
2. Complete the normal purchase flow (it will send you to wordpress.com to make the purchase, which is fine)
3. Copy the url on the thank-you page, and replace the https://wordpress.com portion with http://calypso.localhost:3000. You should end up with the pattern: http://calypso.localhost:3000/checkout/thank-you/{site_slug}.wordpress.com/{receipt_number}
4. You'll see something like the screenshots above

Original design by @mrheston:
![gt-5](https://cloud.githubusercontent.com/assets/416133/16407806/f324da80-3d58-11e6-9d71-d31dd9154109.png)

cc @dllh @omarjackman 

Test live: https://calypso.live/?branch=add/guided-transfer/completion-screen
